pkg/clusterapi/system.go

```diff
@@ -172,7 +172,19 @@ func (c *system) Run(ctx context.Context, installConfig *installconfig.InstallCo
 			),
 		)
 	case gcp.Name:
-		// TODO
+		controllers = append(controllers,
+			c.getInfrastructureController(
+				&GCP,
+				[]string{
+					"-v=2",
+					"--metrics-bind-addr=0",
+					"--health-addr={{suggestHealthHostPort}}",
+					"--webhook-port={{.WebhookPort}}",
+					"--webhook-cert-dir={{.WebhookCertDir}}",
+				},
+				map[string]string{},
```

Comment:
Do you want to add an envvar with hardcoded value to ~/.gcp/osServiceAccount.json and a TODO comment? The JIRA story for auth has already been created and assigned to a future sprint.